error.go

```diff
@@ -6,13 +6,23 @@ type ErrorItem struct {
 	Error string `json:"error"`
 }
 
+type Message struct {
+	Code    int    `json:"code"`
+	Path    string `json:"path"`
+	Message string `json:"message"`
+}
+
 // Error is a structured response indicating a non-2xx HTTP response.
 type Error struct {
-	Errors    ErrorItem `json:"errors"`
+	Warnings  []Message `json:"warnings"`
+	Errors    []ErrorItem `json:"errors"`
 	Reference string    `json:"reference"`
 	Status    int       `json:"status"`
 }
 
 func (e *Error) Error() string {
-	return fmt.Sprintf("voicebase: status=%d %s: %s", e.Status, e.Reference, e.Errors.Error)
+	if len(e.Errors) > 0 {
+		return fmt.Sprintf("voicebase: status=%d %s: %s", e.Status, e.Reference, e.Errors[0].Error)
+	}
+	return ""
```

Comment:
We should probably return something here, at least the status code but likely also reference just in case since an empty error message will be more confusing.